src/components/header/renderer.tsx

```diff
@@ -418,6 +433,56 @@ class Header extends React.Component<Props, State> {
       </div>
     );
 
+    const exportContent = closePortal => (
+      <div className="export-content">
+        <h2>Export</h2>
+        <div className="export-buttons">
+          <button
+            className="export-button"
+            onClick={() => {
+              this.exportViz('png');
+              closePortal();
+            }}
+          >
+            <div>
+              <Image />
+              <span>PNG</span>
+            </div>
+            <p>
+              PNG (Portable Network Graphics) is a raster or bitmap image file format which is made up of a fixed number
+              of pixels that form a complete image.
+            </p>
+          </button>
+          <button
+            className="export-button"
+            onClick={() => {
+              this.exportViz('svg');
+              closePortal();
+            }}
+          >
+            <div>
+              <Map />
+              <span>SVG</span>
+            </div>
+            <p>
+              SVG (Scalable Vector Graphics) is a vector image file format which uses geometric forms to represent
+              different parts of the image as discrete objects.
+            </p>
+          </button>
+        </div>
+        <div className="user-notes">
+          <h4>Notes</h4>
+          <ol>
+            <li>To get a PDF, export SVG followed by saving as PDF from the print window provided by browser.</li>
```

Comment:
To get a PDF, you can export and SVG which you save as PDF from the print window of your browser.